modules/sc-mesh-secure-deployment/src/2_0/features/jamming/util.py

```diff
@@ -0,0 +1,238 @@
+import pathlib
+import random
+import re
+import subprocess
+from enum import Enum
+from typing import Tuple
+
+import netifaces
+import numpy as np
+import pandas as pd
+
+from options import Options
+args = Options()
+
+CH_TO_FREQ = {1: 2412, 2: 2417, 3: 2422, 4: 2427, 5: 2432, 6: 2437, 7: 2442, 8: 2447, 9: 2452, 10: 2457, 11: 2462,
+              36: 5180, 40: 5200, 44: 5220, 48: 5240, 52: 5260, 56: 5280, 60: 5300, 64: 5320, 100: 5500, 104: 5520,
+              108: 5540, 112: 5560, 116: 5580, 120: 5600, 124: 5620, 128: 5640, 132: 5660, 136: 5680, 140: 5700,
+              149: 5745, 153: 5765, 157: 5785, 161: 5805}
+FREQ_TO_CH = {v: k for k, v in CH_TO_FREQ.items()}
```

Comment:
can be calculated from channel number.

Reply:
We statically map the channel to frequency and vice versa as it is a more computationally efficient way to retrieve the required mapping as compared to calculating the corresponding frequency every time it is required. Let me know if this addresses your comment